frontend/src/components/workplan/TestSectionSelectForm.js

```diff
@@ -20,7 +22,18 @@ function TestSectionSelectForm(props) {
 
   useEffect(() => {
     mounted.current = true;
-    getFromOpenElisServer("/rest/user-test-sections", getTestUnits);
+    let testSectionId = new URLSearchParams(window.location.search).get(
```

Comment:
rename searchParam to  `testSectionId` to be consisted with the old UI

Reply:
Thanks, updated to testSectionId